Frontend/src/Components/Graph/GraphSelectModal.js

```diff
@@ -16,10 +16,13 @@ import {
   SliderThumb,
   SliderTrack,
   VStack,
+  useColorMode
 } from "@chakra-ui/react";
-import { useState } from "react";
+import { useEffect, useState } from "react";
```

Comment:
```suggestion
import { useState } from "react";
```
Can remove unused import

Reply:
fixed